src/lib/equinoxe/equinoxe_intf.ml

```diff
@@ -173,12 +173,55 @@ module type API = sig
   end
 end
 
+module type FRIENDLY_API = sig
+  (** It offers OCaml types to manipulate the Equinix API. *)
+
+  type 'a io
+
+  type t
+  (** Abstract type [t] represents the information known by the API system. *)
+
+  val create : ?address:string -> ?token:string -> unit -> t
+  (** [create ~address ~token ()] returns an {!t} object, you need to manipulate
+      when executing requests. Default [address] is
+      [https://api.equinix.com/metal/v1/] and default [token] is empty. *)
+
+  module Orga : sig
+    type config = {
+      name : string;
+      id : string;
```

Comment:
There are a bunch of different ids in the Equinix API (orga, projects, devices, ...). This is not very important for organization, more a note for later: Perhaps we could keep the different `id` types private and separate to clarify the API?

```ocaml
type id
type config = { id : id ; ... }
val delete : Orga.id -> unit io
```

Reply:
Indeed, that's a good idea as it will avoid giving the wrong `id` when we will need it.